src/components/utils/useActionHandlers.ts

```diff
@@ -15,10 +15,8 @@ interface UseActionHandlersResult<T> {
 export function useActionHandlers<T>(callback?: AnyFunction): UseActionHandlersResult<T> {
     const onKeyDown = React.useCallback(
         (event: React.KeyboardEvent<T>) => {
-            if (['Enter', ' ', 'Spacebar'].includes(event.key)) {
-                event.preventDefault();
-
-                return callback?.(event);
+            if (callback && ['Enter', ' ', 'Spacebar'].includes(event.key)) {
+                return callback(event);
```

Comment:
If callback exists should we use `event.preventDefault`?

Reply:
Don't think so, this hook emulates handling click in controls like `<button>`, they didn't prevent events